src/static/riot/queues/list.tag

```diff
@@ -0,0 +1,282 @@
+            self.selected_queue = queue
+            $('.collab-search')
+                .dropdown('setup menu', {values: queue.organizers})
+                .dropdown('set selected',  _.map(queue.organizers, o => o.username))
+
+            if (queue.is_public) {
+                self.refs.queue_public.checked = true
+            }
+        }
+
+        self.handle_queue = function () {
+            let data = {
+                name: self.refs.queue_name.value,
+                is_public: self.refs.queue_public.checked,
+                organizers: _.compact($('.collab-search').dropdown('get value').split(','))
+            }
+            let endpoint = !_.isEmpty(self.selected_queue)
+                ? CODALAB.api.update_queue
+                : CODALAB.api.create_queue
+            endpoint(data, _.get(self, 'selected_queue.id'))
+                .done(function (response) {
+                    toastr.success("Succesfully updated/made queue!")
+                    self.close_modal()
+                    self.update_queues()
+                })
+                .fail(function (response) {
+                    toastr.error("Could not update/create queue!")
+                })
+        }
+
+        /*---------------------------------------------------------------------
+         Table Methods
+         ---------------------------------------------------------------------*/
+        self.filter = function (filters) {
+            filters = filters || {}
+            _.defaults(filters, {
+                search: $(self.refs.search).val(),
+                page: 1,
+            })
+            self.page = filters.page
+            self.update_queues(filters)
+        }
+
+        self.next_page = function () {
+            if (!!self.pagination.next) {
+                self.page += 1
+                self.filter({page: self.page})
+            } else {
+                alert("No valid page to go to!")
+            }
+        }
+        self.previous_page = function () {
+            if (!!self.pagination.previous) {
+                self.page -= 1
+                self.filter({page: self.page})
+            } else {
+                alert("No valid page to go to!")
+            }
+        }
+
+        self.delete_queue = function (queue) {
+            if (confirm("Are you sure you want to delete '" + queue.name + "'?")) {
+                CODALAB.api.delete_queue(queue.id)
+                    .done(function () {
+                        self.update_queues()
+                        toastr.success("Queue deleted successfully!")
+                    })
+                    .fail(function () {
+                        toastr.error("Could not delete queue!")
+                    })
+            }
+            event.stopPropagation()
+        }
+
+        self.copy_queue_url = function (queue) {
+            navigator.clipboard.writeText(queue.broker_url).then(function () {
+                /* clipboard successfully set */
```

Comment:
change these to be line comments, not block?

Reply:
Updated